test/applications/cmd/icmp-responder-nse/main.go

```diff
@@ -103,7 +104,7 @@ func main() {
 		logrus.Fatalf("%v", err)
 	}
 
-	monitorServer = nsmEndpoint.MonitorServer()
+	monitorServer = endpoints[0].GetOpaque(nil).(local.MonitorServer)
```

Comment:
@nickolaev I would prefer to have 
```go
monitorEndpoint := endpoint.NewMonitorEndpoint(nil)
endpoints := []endpoint.ChainedEndpoint{
    monitorEndpoint,
}
```
And when use it here.

Reply:
Done, thanks!